src/context.ts

```diff
@@ -53,6 +54,7 @@ export async function fetchContext(
     event: rest,
     workflowRun: mergedWorkflowRunContext,
     jobRun: mergedJobRunContext,
+    ...(additionalInformation && { additionalInformation })
```

Comment:
This is an optional parameter anyway, we should just pass it in.
```suggestion
    additionalInformation 
```